Formula/golines.rb

```diff
@@ -0,0 +1,27 @@
+class Golines < Formula
+  desc "Golang formatter that fixes long lines"
+  homepage "https://github.com/segmentio/golines"
+  url "https://github.com/segmentio/golines/archive/v0.11.0.tar.gz"
+  sha256 "d7336fbddb045bd2448629c4b8ef5ab2dc6136e71a795b6fdd596066bc00adc0"
+  license "MIT"
+
+  depends_on "go"
```

Comment:
It should be sufficient to make it a build dependency:

```suggestion
  depends_on "go" => :build
```